src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -226,7 +226,10 @@ public class Thread implements Runnable {
         registerNatives();
     }
 
-    /* Reserved for exclusive use by the JVM, maybe move to FieldHolder */
+    /* Reserved for exclusive use by the JVM. Cannot be moved to FieldHolder
+       as it needs to be set by the VM before executing the constructor that
+       will set FieldHolder.
```

Comment:
"will set FieldHolder" is a bit confusing. It creates the FieldHolder and sets the field "holder" so I think the comment could be a bit clearer.

Reply:
Thanks for looking at this @AlanBateman . I have updated the comment.